client/pages/index.jsx

```diff
@@ -1,13 +1,118 @@
-import React from "react";
 import { Head } from "../components";
-import { Button } from "reactstrap";
+import ReactPlayer from "react-player";
+import {
+  Button,
+  Card,
+  CardGroup,
+  CardImg,
+  Col,
+  Container,
+  Input,
+  Row
+} from "reactstrap";
+
+import "../public/style.scss";
+
+const populateImages = () => {
+  const images = new Array(19);
```

Comment:
Magic numbers, I feel like a 126 code mod lol im sorry

Reply:
oops i did this one one sec